### ISSUE TITLE
[❓]: How to cage modded pets? (with right-clicking conflicts)

### ISSUE THREAD

**Issue by SirLuizt** [MehVahdJukaar/Supplementaries#1125]
### Your Question

How to cage modded pets with right-clicking conflicts?

### Detail

I'm using the 1.16.5 version and I know it won't receive any more updates. However, I would like to know if it's possible to create a datapack or something similar to change the right-click action of the cage to another keybind to make it work for mobs like crows from Alex's Mobs or wolves from Revamped Wolves.

The cage works fine when the mobs are untamed (I only tested it with the crows), but it won't work when they are tamed. I assume this occurs due to a right-click conflict, but I haven't been able to find a solution.

**Comment by MehVahdJukaar** [comment]
i dont remember. there definitely is confuguration but that stuff changed so much and that version os so old so i dont know which way it used. check the configs

**Comment by SirLuizt** [comment]
Yup I already checked the config file. The only configuration associated with the cage I found was this:
```
[blocks.cage]
		#Allows all baby mobs to be captured by cages
		cage_allow_all_babies = false
		#Health percentage under which mobs will be allowed to be captured by cages and jars. Leave at 100 to accept any health level
		#Range: 1 ~ 100
		health_threshold = 100
		#Allows all entities to be captured by cages and jars. Not meant for survival
		allow_all_mobs = true
		#Dynamically allows all small mobs inside cages depending on their hitbox size
		cage_auto_detect = false
		
```
But, even after enabling allow_all_mobs, it still doesn't allow me to capture pets in the cage.

**Comment by MehVahdJukaar** [comment]
the way it worked with pets might be the current one, where whatever you do, if its tameable and not tamed it will never let you catch it

**Comment by SirLuizt** [comment]
From what I remember it didn't worked with untamed eagles like you said, but it worked with untamed wolves and crows. So, is there anything that I can do?
